lib/proxy_api/tftp.rb

```diff
@@ -38,12 +39,15 @@ def fetch_boot_file(args)
       raise ProxyException.new(url, e, N_("Unable to fetch TFTP boot file"))
     end
 
-    # returns the TFTP boot server for this proxy
+    # returns the TFTP boot server IP for this proxy
     def bootServer
```

Comment:
I actually dislike doing it here. This class is supposed to just model the proxy API and not do anything else.